### PR TITLE
A6020: Cleanup device tree

### PR DESCRIPTION
**Pull request by karthick111**
Signed-off-by: karthick111 <karthickrko61@gmail.com>

### CODE REVIEW

**Comment by karthick111** [comment]
@skritchz

**Comment by skritchz** [comment]
I was looking at the PR but don't have time to look into sources.
You removed $(call inherit-product-if-exists, vendor/lenovo/A6020/A6020-vendor.mk)

Is this automatically added by some sub inclusions?

**Comment by karthick111** [comment]
removed because it was duplicated
https://github.com/karthick111/android_device_lenovo_A6020/blob/889e1d2c3c10dcc9c8840f2a691be5254734bbe9/full_A6020.mk#L29

File: recovery.fstab [hunk 1]
```diff
@@ -1,10 +0,0 @@
-/system       ext4          /dev/block/platform/soc.0/by-name/system
```
Comment (Shr3ps) [inline recovery.fstab:1]: This file should be there for Lineage stock Recovery

File: system.prop [hunk 2]
```diff
@@ -1,6 +1,40 @@
-#
```
Comment (Shr3ps) [inline system.prop:1]: I would like to keep thoses comments. It helps to have a separation between what is system generated and constant from the device tree.

File: recovery.fstab (hunk 1 above)
Comment (karthick111) [reply recovery.fstab:1]: but nothing mentioned about it in BoardConfig.mk

File: system.prop (hunk 2 above)
Comment (karthick111) [reply system.prop:1]: let @skritchz  decide about it

File: recovery.fstab (hunk 1 above)
Comment (karthick111) [reply recovery.fstab:1]: https://github.com/boulzordev/android_device_lenovo_A6020/blob/cm-14.1/BoardConfig.mk#L119
check this

**Comment by karthick111** [comment]
@skritchz  did u check them ?

File: system.prop (hunk 2 above)
Comment (skritchz) [reply system.prop:1]: I understand what Shr3ps is saying. It's only the first comment that he wants to keep in order to grep in into the build.prop file once generated on the target.

could you just readd `# system.prop for A6020` on the first line. Just for debugging purposes. I'll take the PR right after.

Thanks a lot for all this, I'm busy right now and don't have time to work on the device.

Comment (karthick111) [reply system.prop:1]: okay
but its not that hard to identify what's system generated one

Comment (karthick111) [reply system.prop:1]: anyways its done

Comment (skritchz) [reply system.prop:1]: thanks

**Pull request merged**